notebooks/2019-11-12_Calendars_and_monthly_data.ipynb

```diff
@@ -0,0 +1,576 @@
+       "       cftime.Datetime360Day(2275, 1, 16, 0, 0, 0, 0, 1, 16),\n",
+       "       cftime.Datetime360Day(2275, 2, 16, 0, 0, 0, 0, 3, 46), ...,\n",
+       "       cftime.Datetime360Day(2299, 9, 16, 0, 0, 0, 0, 5, 256),\n",
+       "       cftime.Datetime360Day(2299, 10, 16, 0, 0, 0, 0, 0, 286),\n",
+       "       cftime.Datetime360Day(2299, 11, 16, 0, 0, 0, 0, 2, 316)], dtype=object)\n",
+       "Coordinates:\n",
+       "  * time      (time) object 2274-12-16 00:00:00 ... 2299-11-16 00:00:00\n",
+       "    model_id  <U10 'HadGEM2-ES'\n",
+       "Attributes:\n",
+       "    bounds:         time_bnds\n",
+       "    axis:           T\n",
+       "    long_name:      time\n",
+       "    standard_name:  time"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "var_list[0].time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<xarray.DataArray 'time' (time: 2412)>\n",
+       "array([cftime.DatetimeNoLeap(2100, 1, 16, 12, 0, 0, 0, 1, 16),\n",
+       "       cftime.DatetimeNoLeap(2100, 2, 15, 0, 0, 0, 0, 3, 46),\n",
+       "       cftime.DatetimeNoLeap(2100, 3, 16, 12, 0, 0, 0, 4, 75), ...,\n",
+       "       cftime.DatetimeNoLeap(2300, 10, 16, 12, 0, 0, 0, 5, 289),\n",
+       "       cftime.DatetimeNoLeap(2300, 11, 16, 0, 0, 0, 0, 1, 320),\n",
+       "       cftime.DatetimeNoLeap(2300, 12, 16, 12, 0, 0, 0, 3, 350)], dtype=object)\n",
+       "Coordinates:\n",
+       "  * time      (time) object 2100-01-16 12:00:00 ... 2300-12-16 12:00:00\n",
+       "    model_id  <U10 'bcc-csm1-1'\n",
+       "Attributes:\n",
+       "    bounds:         time_bnds\n",
+       "    axis:           T\n",
+       "    long_name:      time\n",
+       "    standard_name:  time"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "var_list[1].time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see the time coordinates aren't datetime64[ns] objects as it is usually the case for DataArray objects. This is because the models don't use the \"real\" calendar. In that case, Xarray defaults to cftime objects. But then, the objects are different depending on the calendar used in the file. As you can see HadGEM2-ES is using a 360_day calendar while BCC_CSM1.1 is using a NoLeap calendar. \n",
+    "\n",
+    "Having 2 calendars means months don't have the same length in both models. In our case, it doesn't matter as we don't care for data at a finer scale than the month. We also don't want to weight the data in anyway by the length of the month. This also means we do not care for the day, hour etc. of the date. Whether the date is 2100/02/15 12:00:00 or 2100/02/27 00:00:00, both date would represent the same data."
```

Comment:
"We also don't want to weight the data in anyway by the length of the month."
Is that meant to be "in any way"?

Reply:
This also means we do not care **about** the day, hour etc. of the date.